scripts/check-coverage.sh

```
@@ -27,6 +27,11 @@ print_success() {
     printf "${green_color}" "$1" >&2
 }
 
+print_warning() {
+    local yellow_color="\e[33;1m%s\e[0m\n"
+    printf "${yellow_color}" "$1" >&2
```

Shellcheck (0.11.0)

[info] Don't use variables in the printf format string. Use printf '..%s..' "$foo".

(SC2059)